config.go

```diff
@@ -337,23 +337,26 @@ func readAdminSecrets(ctx context.Context, paramMgr *parameterstore.ParameterMan
 					// Create a new map to store the paths
 					newMap := reflect.MakeMap(fieldValue.Type())
 					for _, key := range fieldValue.MapKeys() {
-						mapFieldPath := fmt.Sprintf("%s[%s]", fieldPath, key.String())
+						mapFieldPath := fmt.Sprintf("%s/%s", fieldPath, key.String())
 						param, err := paramMgr.Get(ctx, mapFieldPath)
 						if err != nil {
-							catcher.Add(errors.Wrapf(err, "Failed to store secret map field '%s' in parameter store", mapFieldPath))
+							catcher.Add(errors.Wrapf(err, "Failed to read secret map field '%s' in parameter store", mapFieldPath))
 							continue
 						} else if len(param) > 0 {
 							// Set the map value to the parameter store value
 							newMap.SetMapIndex(key, reflect.ValueOf(param[0].Value))
-						} else {
-							catcher.Add(errors.Errorf("no value found for map key '%s' in parameter store", mapFieldPath))
 						}
 					}
 					// Update the struct field with the new map containing paths
 					if len(newMap.MapKeys()) == len(fieldValue.MapKeys()) {
 						fieldValue.Set(newMap)
 					} else {
-						catcher.Add(errors.New("not all map keys were found in parameter store"))
+						grip.Debug(message.Fields{
```

Comment:
Is this meant as a temporary debug log or is it meant to be here long-term? If it's meant to be a long-term thing, we would likely want to upgrade this to an error/critical since it means there's something wrong with the admin settings.